src/System.Management.Automation/namespaces/NavigationProviderBase.cs

```diff
@@ -648,7 +648,7 @@ protected virtual string GetParentPath(string path, string root)
                     if (!String.IsNullOrEmpty(commonBase))
                     {
                         if (String.Equals(normalizedPath, commonBase, StringComparison.OrdinalIgnoreCase) &&
-                            (!normalizedPath.EndsWith(StringLiterals.DefaultPathSeparatorString, StringComparison.OrdinalIgnoreCase)))
```

Comment:
is `DefaultPathSeparatorString` still used?

Reply:
There are still two links left.

Reply:
thanks